app/src/main/java/com/emmanuelkehinde/twittasave/activities/MainActivity.kt

```diff
@@ -135,15 +136,15 @@ class MainActivity : AppCompatActivity() {
         val sharedText = intent.getStringExtra(Intent.EXTRA_TEXT)
         if (sharedText != null) {
             try {
-                if (sharedText.split("\\ ".toRegex()).dropLastWhile { it.isEmpty() }
+                if (sharedText.split(" ".toRegex()).dropLastWhile { it.isEmpty() }
```

Comment:
I know the IDE complains about this, but I don't trust that the split would be perfect. I would do a test though 😀

Reply:
I'll just undo it to be safe lol